public/app/plugins/datasource/graphite/datasource.ts

```diff
@@ -177,22 +177,24 @@ export class GraphiteDatasource extends DataSourceApi<GraphiteQuery, GraphiteOpt
         maxDataPoints: 100,
       } as unknown) as DataQueryRequest<GraphiteQuery>;
 
-      return this.query(graphiteQuery).then((result: { data: any[] }) => {
+      return this.query(graphiteQuery).then((result: { data: DataFrame[] }) => {
```

Comment:
The result typing here can be completely removed :) The query method has a result type of Promise<DataQueryResponse>  so the result type here can be inferred